dali/test/python/test_fw_iterators.py

```diff
@@ -253,18 +256,91 @@ def test_mxnet_iterator_not_fill_last_batch_pad_last_batch():
     assert len(next_img_ids_list_set) == data_size
     assert len(set(next_mirrored_data)) == 1
 
-def check_mxnet_iterator_pass_reader_name(shards_num, pipes_number, batch_size, stick_to_shard, pad, iters, fill_last_batch):
+LastBatchPolicy = None
+def check_iterator_results(pad, pipes_number, shards_num, out_set, last_batch_policy, img_ids_list,
+                           ids, data_set_size, sample_counter, per_gpu_counter, stick_to_shard,
+                           epoch_counter, rounded_shard_size):
+
+    if pad and pipes_number == shards_num:
+        assert len(set.intersection(*out_set)) == 0, "Shards should not overlaps in the epoch"
+    if last_batch_policy == LastBatchPolicy.DROP:
```

Comment:
How is that supposed to work after LastBatchPolicy has been set to None?

Reply:
I have added just ```
from nvidia.dali.plugin.base_iterator import LastBatchPolicy as LastBatchPolicy```

But I don't know if we want to encourage users to use this internal, undocumented base module.